src/components/CustomTable/CustomTable.types.ts

```diff
@@ -0,0 +1,40 @@
+import { ChangeEvent, MouseEvent } from 'react';
+import { ReactElement } from 'react';
+import { SortingDirection } from '../../common/utils/SortingDirection';
+
+export interface CustomTableToolbarProps<T extends Record<PropertyKey, unknown>> {
+  selected: readonly string[];
+  allRows: readonly T[];
+}
+
+export interface CustomTableHeadProps<T extends Record<PropertyKey, unknown>> {
+  numSelected: number;
+  onRequestSort: (event: MouseEvent<unknown>, property: keyof T) => void;
+  onSelectAllClick: (event: ChangeEvent<HTMLInputElement>) => void;
+  order: SortingDirection;
+  orderBy: string;
+  rowCount: number;
+  // onSortCallback: (headCellName: string, direction: SortingDirection) => void; // will be used later, when we synchronize lists with backend
+}
```

Comment:
If now it is not needed just get rid of this

Reply:
Removed.